### PR TITLE
Update rdm_es_ES to 2019.4

### PR DESCRIPTION
**Pull request by banakito**

### CODE REVIEW

File: src/resources/translations/rdm_es_ES.ts [hunk 1]
```diff
@@ -800,8 +800,8 @@
     </message>
     <message>
         <location filename="../../qml/ConnectionSettignsDialog.qml" line="339"/>
-        <source>&lt;b&gt;Tip:&lt;/b&gt; Use &lt;code&gt;⌘ + Shift + .&lt;/code&gt; to show hidden files and folders in dialog</source>
-        <translation>&lt;b&gt;Tip:&lt;/b&gt; Use &lt;code&gt;⌘ + Shift + .&lt;/code&gt; para mostrar ficheros y carpetas ocultas en el diálogo</translation>
+        <source>&lt;b&gt;Tip:&lt;/b&gt; Use &lt;code&gt;? + Shift + .&lt;/code&gt; to show hidden files and folders in dialog</source>
```
Comment (uglide) [inline src/resources/translations/rdm_es_ES.ts]: Please revert this change (⌘ char is corrupted)

**Review by uglide — CHANGES_REQUESTED** [review]

File: src/resources/translations/rdm_es_ES.ts (hunk 1 above)
Comment (banakito) [reply src/resources/translations/rdm_es_ES.ts]: Fixed

**Comment by uglide** [comment]
Thanks!
